mythril/laser/ethereum/natives.py

```diff
@@ -192,6 +196,19 @@ def ec_pair(data: List[int]) -> List[int]:
     return [0] * 31 + [1 if result else 0]
 
 
+def blake2b_fcompress(data: List[int]) -> List[int]:
```

Comment:
are there ethereum consensus tests that we could add to check if the implementation is correct?

Reply:
There are state tests, but we need to do something with logs to add them. As they check against logs.

Reply:
Also, I have added the fork tests in the tests. So the implementation is correct

Reply:
Right I was more wondering about the integration part of it all. To check if the delegation code passes arguments correctly to the function.
The tests that ckeck blake2b directly are nice though 👍

Reply:
I think it tests the delegation trail